test/view/terminalview.cpp

```diff
@@ -11,19 +11,19 @@ TEST_CASE("A string table can be created"){
         "Food:\n"
         "—————————————————————————————————————————\n"
         "X:  500 | Y:  689 | Energy:  10\n"
-        "X: 1000 | Y: 1000 | Energy:  10\n"
```

Comment:
most of these changes should be done separately

Reply:
these changes shouldn't be there and I should've never touched this test